src/java/org/apache/cassandra/repair/autorepair/AutoRepairKeyspace.java

```diff
@@ -0,0 +1,86 @@
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.cassandra.repair.autorepair;
+
+import java.util.concurrent.TimeUnit;
+
+import org.apache.cassandra.cql3.statements.schema.CreateTableStatement;
+import org.apache.cassandra.schema.KeyspaceMetadata;
+import org.apache.cassandra.schema.KeyspaceParams;
+import org.apache.cassandra.schema.SchemaConstants;
+import org.apache.cassandra.schema.TableId;
+import org.apache.cassandra.schema.TableMetadata;
+import org.apache.cassandra.schema.Tables;
+
+public class AutoRepairKeyspace
+{
+    private AutoRepairKeyspace()
+    {
+    }
+
+    /**
+     * Generation is used as a timestamp for automatic table creation on startup.
+     * If you make any changes to the tables below, make sure to increment the
+     * generation and document your change here.
+     */
+    public static final long GENERATION = 0;
+
+    public static final String AUTO_REPAIR_HISTORY = "auto_repair_history";
+
+    public static final String AUTO_REPAIR_PRIORITY = "auto_repair_priority";
+
+    private static final TableMetadata AutoRepairHistory =
+    parse(AUTO_REPAIR_HISTORY,
+          "Auto repair history for each node",
+          "CREATE TABLE %s ("
+          + "pid int,"
+          + "host_id uuid,"
+          + "repair_type text,"
+          + "repair_turn text,"
+          + "repair_start_ts timestamp,"
+          + "repair_finish_ts timestamp,"
+          + "delete_hosts set<uuid>,"
+          + "delete_hosts_update_time timestamp,"
+          + "force_repair boolean,"
+          + "PRIMARY KEY ((pid, repair_type), host_id))");
+
+    private static final TableMetadata AutoRepairPriority =
+    parse(AUTO_REPAIR_PRIORITY,
+          "Auto repair priority for each group",
+          "CREATE TABLE %s ("
+          + "pid int,"
+          + "repair_type text,"
+          + "repair_priority set<uuid>,"
+          + "PRIMARY KEY ((pid, repair_type)))");
+
+    private static TableMetadata parse(String name, String description, String schema)
+    {
+        return CreateTableStatement.parse(String.format(schema, name), SchemaConstants.AUTO_REPAIR_KEYSPACE_NAME)
+                                   .id(TableId.forSystemTable(SchemaConstants.AUTO_REPAIR_KEYSPACE_NAME, name))
+                                   .comment(description)
+                                   .gcGraceSeconds((int) TimeUnit.DAYS.toSeconds(90))
+                                   .build();
+    }
+
+    public static KeyspaceMetadata metadata()
+    {
+        Tables tables = Tables.of(AutoRepairHistory, AutoRepairPriority);
+        return KeyspaceMetadata.create(SchemaConstants.AUTO_REPAIR_KEYSPACE_NAME, KeyspaceParams.simple(1), tables);
```

Comment:
Should do something similar we did for other system keyspaces in `CassandraRelevantProperties`:

```java
    SYSTEM_AUTH_DEFAULT_RF("cassandra.system_auth.default_rf", "1"),
    SYSTEM_TRACES_DEFAULT_RF("cassandra.system_traces.default_rf", "2"),
    SYSTEM_DISTRIBUTED_DEFAULT_RF("cassandra.system_distributed.default_rf", "3"),
```

which is used like:

```
KeyspaceParams.simple(Math.max(DEFAULT_RF, DatabaseDescriptor.getDefaultKeyspaceRF()))
```

Reply:
although, i wonder if we should just leverage `system_distributed` instead of creating a separate keyspace? This may reduce some operator burden.  e.g. in our cluster bring up one of the first things we do is alter these keyspaces to network topology strategy and we also have allowlist/blocklist for system keyspaces that we'd have to update.

Reply:
Done. Moved these two tables under `system_distributed` itself.